bundle/config/mutator/python/python_mutator.go

```diff
@@ -224,9 +225,10 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 			return dyn.InvalidValue, mutateDiagsHasError
 		}
 
-		newRoot, result, err := applyPythonOutput(leftRoot, rightRoot)
+		newRoot, result0, err := applyPythonOutput(leftRoot, rightRoot)
+		result = result0
 		if err != nil {
-			return dyn.InvalidValue, err
+			return dyn.InvalidValue, fmt.Errorf("failed to apply python output: %w", err)
```

Comment:
nit: I'm not sure if "apply python output" is clear error message for the user.  Perhaps we could have something like "Invalid output from Python mutator bla/bla.py: <error details>"?

Reply:
In general, if this error happens we have a bug, and are not expected to be surfaced to users, so error messages are only needed for debugging.

Reply:
but if it does happen, the users will see this, right?

Regarding the chance of this happening - given that it depends on user input, it seems plausible that there is an output from python subprocess that will trigger this error.

Reply:
It's quite possible that there is/will be a bug, but users cannot do much about it, because we take care of ser/de and merging. So it should be clear that there is nothing wrong about users code, it's a problem in our internals.

Reply:
okay, how about  "internal error when merging output of Python mutator abc/cde.py".

Reply:
Fixed